docs/pages/reference/architecture/architecture.mdx

```diff
@@ -1,10 +1,131 @@
 ---
-title: Teleport Architecture Guides
-description: Get detailed information about how Teleport works
+title: Teleport Architecture
+description: Provides detailed information about how Teleport works.
 ---
 
-In this section, you will find detailed information about Teleport's internal
-architecture. Read these guides if you are interested in learning how Teleport
-works.
+This guide explains the technical architecture of Teleport. Before reading this
+guide, we recommend that you read the [Core Concepts](../../core-concepts.mdx)
+page, which describes the components of a Teleport cluster.
 
-(!toc!)
+## Teleport control plane
+
+Teleport consists of a **control plane** that runs the Teleport Auth Service and
+Teleport Proxy Service, and a **data plane** that consists of Teleport Agents,
+Machine ID Bots, and Auth Service API clients like the Event Handler and Access
+Request plugins. The data plane communicates with both your infrastructure and
+the control plane. Your infrastructure communicates only with the data plane.
```

Comment:
I'm not sure I find the distinction of the "data plane" helpful here.

Teleport agents run on the customer infrastructure, but they absolutely communicate with the control plane.

I think we can simplify this whole section by just defining the term _control plane_ as the auth+proxy services.